packages/salesforcedx-sobjects-faux-generator/test/integration/config/simpleObjectAndField/objects/MyCustomObject2__c/MyCustomObject2__c.object-meta.xml

```diff
@@ -52,7 +52,6 @@
     <enableSearch>false</enableSearch>
     <enableSharing>true</enableSharing>
     <enableStreamingApi>true</enableStreamingApi>
-    <externalSharingModel>ReadWrite</externalSharingModel>
```

Comment:
@GaryFNC - I'm removing this since the externalSharingModel is now being turned off for all scratch orgs. This means that we can't deploy a custom object if we have this set here.